clients/src/main/java/org/apache/kafka/common/protocol/ApiKeys.java

```diff
@@ -79,6 +84,46 @@ public static boolean hasId(int id) {
         return id >= MIN_API_KEY && id <= MAX_API_KEY;
     }
 
+    public short latestVersion() {
+        if (id >= Protocol.CURR_VERSION.length)
```

Comment:
If we're checking for id >= Protocol.CURR_VERSION.length, should we also check for id < 0?